tools/cargo-compare/src/check.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) The cargo-guppy Contributors
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::common::GuppyCargoCommon;
+use crate::GlobalContext;
+use anyhow::{Error, Result};
+use diffus::{edit, Diffable};
+use guppy::PackageId;
+use guppy_cmdlib::CargoMetadataOptions;
+use proptest::prelude::*;
+use proptest::test_runner::{Config, TestCaseResult, TestError, TestRunner};
+use std::collections::{BTreeMap, BTreeSet};
+use std::sync::atomic::{AtomicUsize, Ordering};
+use structopt::StructOpt;
+
+/// Options for cargo/guppy comparisons.
+#[derive(Debug, StructOpt)]
+pub struct CheckOpts {
+    /// Number of randomly generated diff operations to run
+    #[structopt(long)]
+    pub cases: u32,
+    #[structopt(flatten)]
+    pub metadata: CargoMetadataOptions,
+}
+
+impl CheckOpts {
+    /// Executes this command.
+    pub fn exec(self, ctx: &GlobalContext) -> Result<()> {
+        let strat = GuppyCargoCommon::strategy(&self.metadata, ctx.graph);
+
+        let mut testrunner = TestRunner::new(Config {
+            cases: self.cases,
+            ..Config::default()
+        });
+
+        // print a message after every n tests
+        let test_count = AtomicUsize::new(0);
+        let print_every = 5;
```

Comment:
* Could you make this an argument with a default of 16, and set to 0 to disable output?
* Instead of resetting the count back to zero, could you instead keep incrementing it and using `% print_every`?